pkgs/development/python-modules/galois/default.nix

```
@@ -11,18 +12,24 @@
 
 buildPythonPackage rec {
   pname = "galois";
-  version = "0.0.32";
-  format = "setuptools";
+  version = "0.1.1";
+  format = "pyproject";
 
   disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner = "mhostetter";
     repo = "galois";
     rev = "refs/tags/v${version}";
-    sha256 = "sha256-+cxRLrfqk3N9pWKCVsTxruZwMYZ5dQyKJRnrb8y+ECM=";
+    sha256 = "sha256-ekbebwsdGkINdOFI+LLdwTRBN6d8mlNmftkj5SNRKNI=";
   };
 
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
+
+  buildInputs = [
```

```suggestion
  nativeBuildInputs = [
```

buildInputs are runtime dependencies, nativeBuildInputs are build time dependencies.

---

Thanks for the catch. I've fixed this